astacus/node/snapshot.py

```diff
@@ -10,6 +10,7 @@
 
 """
 
+from ..common.rohmustorage import RohmuStorage
```

Comment:
This should probably be an absolute import (not blocking)

Reply:
Good catch, I missed it in the PR split, the next one fixes it.